config/config.go

```diff
@@ -148,3 +150,27 @@ func isSupportedBenchmark(benchmark string) bool {
 	}
 	return false
 }
+
+// TODO: move config validation to libs? (aws/gcp)
+// TODO: why is "" a valid case?
+func isSupportedAccountType(c Config) error {
+	var collection = []string{
+		"", SingleAccount, OrganizationAccount,
+	}
+
+	if !lo.Contains(collection, c.CloudConfig.Aws.AccountType) {
+		return launcher.NewUnhealthyError(fmt.Sprintf(
+			"aws.account_type '%s' is not supported",
+			c.CloudConfig.Aws.AccountType,
+		))
+	}
+
+	if !lo.Contains(collection, c.CloudConfig.Gcp.AccountType) {
+		return launcher.NewUnhealthyError(fmt.Sprintf(
+			"gcp.account_type '%s' is not supported",
+			c.CloudConfig.Gcp.AccountType,
+		))
+	}
```

Comment:
I'm not sure it's needed as we already perform this kind of validation in the credentials BL.